qiita_pet/templates/private_studies.html

```diff
@@ -88,6 +88,7 @@ <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
         <thead>
             <tr>
                 <th>Title</th>
+                <th>Study ID</th>
```

Comment:
Also need to add this code to the user studies table. This only adds it to the shared studies table.